client/src/Molecules.elm

```diff
@@ -37,50 +44,82 @@ type Bsa1Overhang
     | E__F
     | E__G
     | F__G
+    | InvalidBsa1
```

Comment:
Why was this added? You should probably use a `Maybe` or `Result` type to handle errors like this.

Reply:
I've resolved this in the next commit.